app/docs/0.10.x/admin-api.md

```diff
@@ -759,7 +760,9 @@ The order of precedence is, from highest to lowest:
 configurations): for an API (Plugin config A), and for a Consumer (Plugin
 config B), then requests authenticating this Consumer will run Plugin config B
 and ignore A (2.). However, requests that do not authenticate this Consumer
-will fallback to running Plugin config A (3.).
+will fallback to running Plugin config A (3.). Note that if config B is
+disabled, config A will apply to requests that would have otherwise matched
```

Comment:
I think we'd better mention _how_ a plugin can be "disabled", or else the reader will be confused. Something like:
```
if config B is disabled (its `enabled` flag is set to `false`), then config A will apply to requests...
```